TFG-Labs/custom-shipping-step-by-items
# Feature/tlva 102/new rica fields

### What problem is this solving?

Functionality developed to satisfy customer requirements for simcard products requiring RICA information.

#### How it works

If there is a product from the simcard category in the shopping cart, the customer will be shown a form that must be filled in.

The data from this form will be stored in the order form of the order.

#### How to test it?

[Workspace](https://tlva102--thefoschini.myvtex.com/)

You must be logged in with a customer already registered on the platform.

In the checkout add the product with SKU **59418100**, which has category RICA.

When you reach the Shipping step and you have entered the shipping address, the form will appear as you can see in the following video

### Screenshots/Video example usage:

[Video](https://www.loom.com/share/80f570256eee457bb753c9db511c7362)

#### How does this PR make you feel? 🔗

![](https://media.giphy.com/media/T8Dhl1KPyzRqU/giphy.gif)
